scripts/create_aad_assets.sh

```diff
@@ -36,7 +36,7 @@ fi
 # Then register an App for the TRE Core.
 ./scripts/aad/aad-app-reg.sh \
   --name "${TRE_ID}" \
-  --swaggerui-redirecturl "https://${TRE_ID}.${LOCATION}.cloudapp.azure.com/api/docs/oauth2-redirect" \
+  --tre-url "https://${TRE_ID}.${LOCATION}.cloudapp.azure.com/api/docs/oauth2-redirect" \
```

Comment:
I think this should just be `https://${TRE_ID}.${LOCATION}.cloudapp.azure.com`, not the full redirect path. As we add that on later in the script for both swagger and the new UI.

Reply:
Yep...

https://github.com/microsoft/AzureTRE/blob/main/scripts/aad/aad-app-reg.sh#L509

... so, `create_aad_assets.sh` was broken in two ways then?

Reply:
Yes, although both on the same line. Expect something has got lost on a merge. I made the changes to the script and docs and ran through a couple of times.

Reply:
@martinpeck I've remove that path. Good to approve?